_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of Victor Ikenna Echendu](https://user-images.githubusercontent.com/47511784/79973631-d2cc1000-848f-11ea-9e0e-538e3cfc935f.jpg)